IdentityCore/src/MSIDOAuth2Constants.h

```diff
@@ -37,6 +37,7 @@ extern NSString *const MSID_OAUTH2_ERROR;
 extern NSString *const MSID_OAUTH2_ERROR_DESCRIPTION;
```

Comment:
This pull request does not update changelog.txt.

Please consider if this change would be noticeable to a partner or user and either update changelog.txt or resolve this conversation.
<!-- changelog-comment -->

Reply:
This is auto generated message that whenever a PR is created and no change logs is updated, so feel free to ignore it if needed

Reply:
Also I would recommend to create a draft pr if was not ready for review :)